Node.cpp

```diff
@@ -51,27 +48,27 @@ CNode& CNode::operator = (const CNode& node)
     return (*this);
 }
 
-void CNode::StoreTree(int nIndent, int nIndex)
+void CNode::StoreTree(std::ofstream& ofstream, int nIndent, int nIndex)

```

Comment:
Maybe better to use more "normal" name like outputStream/inputStream (or just `stream`)?

Reply:
Sure, i tried to follow current naming (same applies to `m_*stream` mentioned above) but no problem refreshing it a bit. I'll get to it tomorrow morning.